### PR TITLE
Changelog for PIX, new versions of Java, .NET and Magento 2

### PR DESCRIPTION
**Pull request by lamaralmp**
## Description
Adding new versions of SDKs Java and .NET in the changelog.
And a new version of Magento 2 plugin with support a the Mercado Pago Wallet button and same fixes

Ese PR reemplaza el #1075 y el #1080

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

**Review by lira — APPROVED** [review]
OK! 👍🏽

File: guides/resources/changelog/current-year.en.md [hunk 1]
```diff
@@ -3,6 +3,80 @@
...
+
+------------
+
+> CHANGELOG
+>
+> New version of SDK Java
+>
+> NEW_VERSION: NEW VERSION
+>
+> PRODUCT: SDK Java
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.en.md:32]:
```suggestion
> PRODUCT: SDK JAVA
```

File: guides/resources/changelog/current-year.es.md [hunk 2]
```diff
@@ -3,6 +3,80 @@
...
+
+------------
+
+> CHANGELOG
+>
+> Nueva versión SDK Java
+>
+> NEW_VERSION: NUEVA VERSIÓN
+>
+> PRODUCT: SDK Java
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.es.md:32]:
```suggestion
> PRODUCT: SDK JAVA
```

File: guides/resources/changelog/current-year.es.md [hunk 3]
```diff
@@ -3,6 +3,80 @@
...
+
+Lanzamos una nueva versión del SDK NodeJS 1.9.0.
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.es.md:34]:
```suggestion
Lanzamos una nueva versión del SDK Java 1.9.0.
```

File: guides/resources/changelog/current-year.es.md [hunk 4]
```diff
@@ -3,6 +3,80 @@
...
+
+> GIT
+>
+> GitHub
+>
+> Para más detalle, consulta en GitHub los [últimos cambios productivos](https://github.com/mercadopago/sdk-nodejs/releases/tag/1.9.0).
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.es.md:40]:
```suggestion
> Para más detalle, consulta en GitHub los [últimos cambios productivos](https://github.com/mercadopago/sdk-java/releases/tag/1.9.0).
```

File: guides/resources/changelog/current-year.es.md [hunk 5]
```diff
@@ -3,6 +3,80 @@
...
+
+> CHANGELOG
+>
+> Nueva versión SDK .NET
+>
+> NEW_VERSION: NUEVA VERSIÓN
+>
+> PRODUCT: SDK .NET
+
+Lanzamos una nueva versión del SDK NodeJS 1.10.0.
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.es.md:50]:
```suggestion
Lanzamos una nueva versión del SDK .NET 1.10.0.
```

File: guides/resources/changelog/current-year.es.md [hunk 6]
```diff
@@ -3,6 +3,80 @@
...
+
+> GIT
+>
+> GitHub
+>
+> Para más detalle, consulta en GitHub los [últimos cambios productivos](https://github.com/mercadopago/sdk-nodejs/releases/tag/1.10.0).
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.es.md:56]:
```suggestion
> Para más detalle, consulta en GitHub los [últimos cambios productivos](https://github.com/mercadopago/dx-dotnet/releases/tag/1.10.0).
```

File: guides/resources/changelog/current-year.pt.md [hunk 7]
```diff
@@ -3,6 +3,80 @@
...
+
+> CHANGELOG
+>
+> Nova versão SDK Java
+>
+> NEW_VERSION: NOVA VERSÃO
+>
+> PRODUCT: SDK Java
+
+Lançamos uma nova versão do SDK NodeJS 1.9.0.
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.pt.md:34]:
```suggestion
Lançamos uma nova versão do SDK Java 1.9.0.
```

File: guides/resources/changelog/current-year.pt.md [hunk 8]
```diff
@@ -3,6 +3,80 @@
...
+
+> GIT
+>
+> GitHub
+>
+> Confira no GitHub o detalhe das [últimas atualizações produtivas](https://github.com/mercadopago/sdk-nodejs/releases/tag/1.9.0).
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.pt.md:40]:
```suggestion
> Confira no GitHub o detalhe das [últimas atualizações produtivas](https://github.com/mercadopago/java/releases/tag/1.9.0).
```

File: guides/resources/changelog/current-year.pt.md [hunk 9]
```diff
@@ -3,6 +3,80 @@
...
+
+> CHANGELOG
+>
+> Nova versão SDK .NET
+>
+> NEW_VERSION: NOVA VERSÃO
+>
+> PRODUCT: SDK .NET
+
+Lançamos uma nova versão do SDK NodeJS 1.10.0.
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.pt.md:50]:
```suggestion
Lançamos uma nova versão do SDK .NET 1.10.0.
```

File: guides/resources/changelog/current-year.pt.md [hunk 10]
```diff
@@ -3,6 +3,80 @@
...
+
+> GIT
+>
+> GitHub
+>
+> Confira no GitHub o detalhe das [últimas atualizações produtivas](https://github.com/mercadopago/sdk-nodejs/releases/tag/1.10.0).
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.pt.md:56]:
```suggestion
> Confira no GitHub o detalhe das [últimas atualizações produtivas](https://github.com/mercadopago/dx-dotnet/releases/tag/1.10.0).
```

**Review by pamalv7 — CHANGES_REQUESTED** [review]

File: guides/resources/changelog/current-year.pt.md [hunk 11]
```diff
@@ -3,6 +3,80 @@
...
+
+------------
+
+> CHANGELOG
+>
+> Nova versão SDK Java
+>
+> NEW_VERSION: NOVA VERSÃO
+>
+> PRODUCT: SDK Java
```
Comment (pamalv7) [inline guides/resources/changelog/current-year.pt.md:32]:
```suggestion
> PRODUCT: SDK JAVA
```

**Review by pamalv7 — APPROVED** [review]

**Pull request merged**